templates/vr-ar.jade

```diff
@@ -88,15 +89,111 @@ block main
+                      td 120Hz
+                    tr
+                      td.Table--column Field of View
+                      td As far as the HMD goes
+                    tr
+                      td.Table--column Gaze Accuracy
+                      td ~1.0deg
+                    tr
+                      td.Table--column Gaze Precision
+                      td ~0.08deg
+                    tr
+                      td.Table--column #[strong Latency]
+                      td 
+                    tr
+                      td.Table--column Camera Latency
+                      td 5.7ms
+                    tr
+                      td.Table--column Processing latency (specify cpu assumptions)
+                      td 3-4ms on 15 CPU
+                    tr
+                      td.Table--column #[strong Imaging]
+                      td 
+                    tr
+                      td.Table--column Resolution
+                      td 640x480
+                    tr
+                      td.Table--column Connection
+                      td USB 2.0
+                    tr
+                      td.Table--column Saturation
+                      td Interconnect bandwidth USB 2.0 ~60% saturation
+                    tr
+                      td.Table--column #[strong Data Output]
+                      td 
+                    tr
+                      td.Table--column Combined Eye Gaze
+                      td Yes
+                    tr
+                      td.Table--column Independent Eye Gaze (1 gaze point per eye)
+                      td Yes
+                    tr
+                      td.Table--column Pupil Position
+                      td Yes
+                    tr
+                      td.Table--column Pupil Size
+                      td Yes
+                    tr
+                      td.Table--column Velocity/Acceleration
+                      td Can be implemented if desired
+                    tr
+                      td.Table--column Eye Accomodation
+                      td Yes, using binocular vergence
+                    tr
+                      td.Table--column #[strong Calibration]
+                      td 
+                    tr
+                      td.Table--column Method
+                      td 5 point
+                    tr
+                      td.Table--column #[strong Power/Performance/Memory]
+                      td 
+                    tr
+                      td.Table--column 
+                      td Host CPU % Utilization: 50% of one core per eye
+                    tr
+                      td.Table--column #[strong Connectivity]
+                      td 
+                    tr
+                      td.Table--column SDK
+                      td Through network message based API
+                    tr
+                      td.Table--column Plugin for Game Engines
+                      td Reference Implementation for Unity3D
```

Comment:
[Reference plugin](https://github.com/pupil-labs/hmd-eyes) implementation for Unity3D